allennlp/commands/train.py

```diff
@@ -172,7 +186,41 @@ def train_model(params: Params, serialization_dir: str, file_friendly_logging: b
     """
     prepare_environment(params)
 
-    os.makedirs(serialization_dir, exist_ok=True)
+    if os.path.exists(serialization_dir):
+        logger.info(f"Continuing from prior training at {serialization_dir}.")
+
+        continued_config_file = os.path.join(serialization_dir, CONFIG_NAME)
+        if not os.path.exists(continued_config_file):
+            logger.warning("Continuing from prior training, but no config.json was found.")
```

Comment:
Again here I'd just crash.  If something unexpected happens, we should fail explicitly instead of silently.

Reply:
It might just be an empty directory.  I'd rather not crash here.  However, I don't like the warning message so I'll improve it.

Reply:
What's the circumstance when you'd have an empty directory that you want to continue training from?  It seems like you can only get to this branch on accident, which is almost certainly a reason to crash, not fail with a warning you might not notice.